addons/event/models/event.py

```diff
@@ -22,6 +22,7 @@
 class EventType(models.Model):
     _name = 'event.type'
     _description = 'Event Category'
+    _order = 'sequence'
```

Comment:
If two records have the same sequence, the result will be nondeterministic. To avoid this issue, it is a good practice to always add `sequence, id` to guarantee a stable order.

Reply:
done